dashboard/app/controllers/levels_controller.rb

```diff
@@ -234,7 +234,8 @@ def level_params
       :dsl_text,
       :encrypted,
       {concept_ids: []},
-      {soft_buttons: []}
+      {soft_buttons: []},
+      {examples: []}
```

Comment:
not in the same way, because it's not a multiselect. Empty/nil values are actually filtered out in applab.rb on read of the property.